api/src/main/java/io/opentelemetry/trace/SpanContext.java

```diff
@@ -44,7 +44,7 @@
    *
    * @return the invalid {@code SpanContext}.
    */
-  static SpanContext getInvalid() {
+  public static SpanContext getInvalid() {
```

Comment:
I remember we didn't want to expose the `SpanContext.getInvalid()` instance, in order to prevent users abusing it. So we either lift that restriction or we use, for the TraceContext propagator, our own INVALID instance (build out from invalid ids).

cc @yurishkuro

Reply:
our own INVALID instance will be the same as ```SpanContext.getInvalid()``` maybe we should move it to same base part were both of SpanContext and other parts of opentelemetry can construct INVALID instance

Reply:
Let's keep our own invalid instance of `SpanContext` for now and create a ticket discussing whether we should keep a public accessor for `INVALID`.